tests/e2e/tools/lib/editor.js

```diff
@@ -76,9 +76,7 @@ async function removeAllPostTypes({editor}) {
  */
 const searchAndInsertBlock = async ({page}, blockName, namespace = '') => {
   const labelText = 'Toggle block inserter'; // Can we also use aria-label="Add block"
-  const addBlockButton = await page.locator(`button[aria-label="${labelText}"]`);
-  expect(addBlockButton).toBeVisible();
```

Comment:
Do you think it is not neccesary this `expect`? If you agree to remove it, please also remove it from [here](https://github.com/greenpeace/planet4-master-theme/blob/5815c22ba747377c172e6f534001e1719ea14ad1/tests/e2e/tools/lib/editor.js)

Reply:
Yeah, as I explained in the corresponding [commit](https://github.com/greenpeace/planet4-master-theme/pull/2420/commits/69040a862d7c883776d5b58d06f44acb31bb4c19), the Playwright 'click' function already makes sure that the element is visible so we don't need to check ourselves

Reply:
I don't understand where else you want me to remove it 🤔

Reply:
Sorry, I've nevera added the line's link

Reply:
In the first line? I don't see what needs to be changed

Reply:
Don't you see `import {expect, Locator, Page} from '@playwright/test'; // eslint-disable-line no-unused-vars` ?

If yes.. please remove the `expect` import, please!

Reply:
Aaaaah, my bad, sorry! I thought `expect` would be used somewhere else in the file